Update content for basic REST service/client samples (#3764)
ballerina-platform/ballerina-distribution: examples/http-client-send-request-receive-response/http_client_send_request_receive_response.md

# HTTP client - Send request/Receive response

> The `http:Client` is used to connect and interact with an HTTP server. The client is instantiated with the service URL and it will use resource methods to interact with the backend service. The standard HTTP methods `get`, `post`, `put`, `patch`, `delete`, `head`, and `options` are available as resource methods. A resource method can be invoked by providing required arguments and the relevant HTTP verb after the `->`. Since HTTP `get` is the default resource method the verb is not mandatory when accessing an HTTP `get` resource.

Author: Fixed with [b870a2f6b2c9eb5af5fb4ca00a5442852595b15c](https://github.com/ballerina-platform/ballerina-distribution/pull/3764/commits/b870a2f6b2c9eb5af5fb4ca00a5442852595b15c)

> The `http:Client` is used to connect and interact with an HTTP server. The client is instantiated with the service URL and it will use resource methods to interact with the backend service. The standard HTTP methods `get`, `post`, `put`, `patch`, `delete`, `head`, and `options` are available as resource methods. A resource method can be invoked by providing required arguments and the relevant HTTP verb after the `->`. Since HTTP `get` is the default resource method the verb is not mandatory when accessing an HTTP `get` resource.

Author: Fixed with [b870a2f6b2c9eb5af5fb4ca00a5442852595b15c](https://github.com/ballerina-platform/ballerina-distribution/pull/3764/commits/b870a2f6b2c9eb5af5fb4ca00a5442852595b15c)

> The `http:Client` is used to connect and interact with an HTTP server. The client is instantiated with the service URL and it will use resource methods to interact with the backend service. The standard HTTP methods `get`, `post`, `put`, `patch`, `delete`, `head`, and `options` are available as resource methods. A resource method can be invoked by providing required arguments and the relevant HTTP verb after the `->`. Since HTTP `get` is the default resource method the verb is not mandatory when accessing an HTTP `get` resource.

Author: Ad discussed will use the current convention.

> The `http:Client` interacts with an HTTP server. The client is instantiated with the service URL and it uses resource methods to send/receive requests/responses from the backend service. The standard HTTP methods `get`, `post`, `put`, `patch`, `delete`, `head`, and `options` are available as resource methods. A resource method can be invoked by providing the `resource path`, relevant resource method, and required arguments after the `->`. Since HTTP `get` is the default resource method, the accessor is not mandatory when accessing an HTTP `GET` resource.

Reviewer: ```suggestion
The `http:Client` interacts with an HTTP server. The client is instantiated with the service URL and uses resource methods to send requests to the backend service. The standard HTTP methods `get`, `post`, `put`, `patch`, `delete`, `head`, and `options` are available as resource accessors. A resource method invocation is done by providing the `resource path`, relevant `resource accessor`, and required arguments after the `->`. Since HTTP `get` is the default resource method, the accessor is not mandatory when invoking an HTTP `GET` resource.
```

Author: Fixed with [ee2c31ed2d6e7244a7858bb4bcebdf2385e21087](https://github.com/ballerina-platform/ballerina-distribution/pull/3764/commits/ee2c31ed2d6e7244a7858bb4bcebdf2385e21087)

> The `http:Client` interacts with an HTTP server. The client is instantiated with the service URL and it uses resource methods to send requests and receive responses from the backend service. The standard HTTP methods `get`, `post`, `put`, `patch`, `delete`, `head`, and `options` are available as resource accessors. A resource method invocation is done by providing the `resource path`, relevant `resource accessor`, and required arguments after the `->`. Since HTTP `get` is the default resource method, the accessor is not mandatory when invoking an HTTP `GET` resource.

Reviewer 2: The `http:Client` interacts with an HTTP server. The client is instantiated with the service URL and it uses resource methods to send requests and receive responses from the backend service. The standard HTTP methods: `get`, `post`, `put`, `patch`, `delete`, `head`, and `options` are available as resource accessors. A resource method invocation is done by providing the `resource path`, relevant `resource accessor`, and required arguments after the `->`. Since HTTP `get` is the default resource method, the accessor is not mandatory when invoking an HTTP `GET` resource.
